examples/model_hosting/notebook/training/LocalTraining.ipynb

```diff
@@ -120,7 +120,7 @@
     "df = data_fetcher.time_series.fetch_dataframe([\"temperature\", \"pressure\", \"friction\"])\n",
     "\n",
     "%matplotlib inline\n",
-    "df.plot(x=\"timestamp\")"
+    "df.plot()"
```

Comment:
Don't need/want `x="timestamp"` anymore?

Reply:
As above, not anymore.